app/models/user.rb

```diff
@@ -1,8 +1,8 @@
 class User < ApplicationRecord
   # Associations
   has_many :post, foreign_key: 'author_id'
-  has_many :comment, foreign_key: 'author_id'
-  has_many :like, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
```

Comment:
- [ ] **[OPTIONAL]** This change is for the Comment model, not this user model. This is a very important change. It is optional, only because it is outside the purview of this project but you should consider fixing it. The Comment is associated with a `user`, which is good and fine, but the Comment table schema contains `author_id`, instead of `user_id`. Kindly run a new migration to rename the table column to `user_id` as required from a previous associations project. Your Like schema is well created, kudos for that.

  ```
  Comment(id: integer, author_id: integer, post_id: integer, text: text, created_at: datetime, updated_at: datetime)
  ```

- [ ] **[OPTIONAL]** After that, this line here needs to be changed to the below to fit the association, by removing the `foreign_key` part. You should also do the same for the `has_many :likes` line.

  ```
  has_many :comments
  ```